core/system/src/mpu/vmpu_armv7m.c

```diff
@@ -629,10 +628,11 @@ void vmpu_arch_init(void)
 {
     /* reset MPU */
     MPU->CTRL = 0;
-    /* check MPU region aligment */
-    MPU->RBAR = MPU_RBAR_ADDR_Msk;
+    /* check MPU region aligment using region 0 */
+    MPU->RNR = 0;
```

Comment:
Do we need to program the RNR to something else later, or is leaving it at 0 okay?